File: C_Sharp_code/Example/WindowsFormsApp/VB_demo16/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim stu As New Student
        stu.No = 201
        stu.Name = "Mary"
        stu.Major = "Chinese"
        Console.WriteLine("{0},{1},{2}", stu.No, stu.Name, stu.Major)
    End Sub
End Class

File: C_Sharp_code/Example/WindowsFormsApp/CS_demo15/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace CS_demo15
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            Student stu = new Student();
            stu.No = 201;
            stu.Name = "Eason";
            stu.Major = "English";
            Console.WriteLine("{0},{1},{2}",stu.No,stu.Name,stu.Major);
        }
    }
}
